README.md

```diff
@@ -77,14 +77,19 @@ In the `./nodezoo-system` folder run,
 fuge build fuge/system.yml
 ```
 
-### 6. Start the infrastructure
+### 6. Insert GitHub token
+For the GitHub service to work you must insert your GitHub personal access token in to the sample.env file located in fuge/env.
```

Comment:
system.env is not checked in, so you can put all your private stuff in it and not accidentally commit.